src/khoj/processor/conversation/prompts.py

```diff
@@ -253,6 +253,7 @@
 **/notes**: Chat using the information in your knowledge base.
 **/general**: Chat using just Khoj's general knowledge. This will not search against your notes.
 **/default**: Chat using your knowledge base and Khoj's general knowledge for context.
+**/clear**: Clear the chat session.
```

Comment:
Hmm, I'm not really a fan of having this in the `/` commands. I think it should be a UI button that allows the user to clear the chat. That will be more extensible as we go into a multi-session design and allow users to clear/manage multiple chats simultaneously. We'll want it to do be a specific API call that deletes that specific conversation by ID. For now, deleting all by the user is ok, but it should be eventually be granular on the specific chat.

`/` commands IMO are more about modality, less about management.

Reply:
Makes sense. Moved to using a delete button, instead of using a slash command to clear conversation